### ISSUE TITLE
Deploy and reindex stage and prod to show edited Scope and Content

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/vt-arclight#469]
There's some invalid HTML rendering in the `ul` in the Background > Scope and Content section of the collection level metadata. This generates an accessibility error. They were apparently added to create a complete sentence form of list, along with the semicolons at the end of each list item.

This is at the top-level of the content inventory: `catalog/mt839rq8746` Scroll to the end of the bulleted list under Scope and Content. There is the word "and" and a period around the last list item:

<img width="615" alt="Screen Shot 2023-01-13 at 4 44 12 PM" src="https://user-images.githubusercontent.com/101482/212438715-0de92b31-f385-4fcf-af08-104f6f57d117.png">

---

Because these are not within an `li`, they are direct children of the `ul` and you can't have text elements as direct children of a `ul`. So the "and" and the period should be removed. Although not necessary to make the list valid, ideally we should also remove the semicolons at the end of each list item as well, since we're no longer trying to make the list a complete sentence. 

I'm guessing this involves editing things in ASpace (not sure of the exact process of generating the HTML), but what we should end changing the list to look like this:

<img width="615" alt="Screen Shot 2023-01-13 at 5 06 34 PM" src="https://user-images.githubusercontent.com/101482/212439746-31f6210a-eb47-46e1-bd71-8a7d4fd3935c.png">

---

(Alternatively, we could leave the semicolons and add the "and" to the end of the next-to-last list item -- after the semicolon -- and the period to the end of the last list item. That would make a valid, complete sentence style list. But I think the first approach makes the list easier to parse and is worth the extra few minutes of editing.)

**Comment by marlo-longley** [comment]
@ggeisler I edited the content in ASpace to your specifications, thanks! I'm changing the title of this ticket to better reflect the work needed to get the change live.
